### DISCUSSION TITLE
How to improve light to make the model brighter or have more light in the AR view

### DISCUSSION THREAD

**Discussion by rianlukog** [google/model-viewer#2533] [Q&A]
Where` do I add the code snippets ?
Code below:

<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Model Viewer</title>
    <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.min.js"></script>
    <style>
model-viewer{
    width: 400px;
    height: 600px;
    margin: 0 auto;
  --poster-color: transperant;

}
    </style>
</head>
<body>
    <model-viewer src="SOURCE" camera-controls auto-rotate ar ios-src="SOURCE2"></model-viewer>
</body>
</html>

**Comment by elalish** [comment]
I'm not sure what you're asking, and your code snippet does not include the `<model-viewer>` element, which is where all the customization happens anyway.
